narayana/scripts/hudson/bm.sh

```diff
@@ -19,10 +19,18 @@ function cmp_narayana {
 
 function runs {
   echo -e "threads\tmaxIO\tbf/s\tAIO\ttput\t\tchange" > results.txt
+  PROGRESS=0
+  INC=100
+  LOOP_ITERATION=16
   for threads in 1 24 240 1600; do
     for bf in 300 2000 4000 7812; do
+      PROGRESS=`expr $PROGRESS + $INC`
+      START_TIME=$(($(date +%s%N)/1000000))
       cmp_narayana $threads 500 $bf true r-${threads}-${bf}.txt
       cmp_narayana $threads 1 $bf false r.txt $tput
+      END_TIME=$(($(date +%s%N)/1000000))
+      EXE_DURATION=`expr $END_TIME - $START_TIME`
+      echo -e "threads=$threads \t bf=$bf \t Benchmark execution duration=$EXE_DURATION milliseconds \t ProgressPercent=`expr $PROGRESS / $LOOP_ITERATION`%" > results.txt
```

Comment:
This line is overwriting the contents of results.txt

Reply:
I will remove this, Thanks